tests/check.sh

```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/bash
+
+# Check if shellcheck is available. If not, disable shellcheck usage.
+command -v shellcheck
```

Comment:
We can't get ShellCheck on RHEL?

![image](https://user-images.githubusercontent.com/10236257/50777708-509fe400-126a-11e9-8523-6095f7621ab9.png)

Reply:
As long as we don't fork the code though, and we have CI coverage that's testing using Fedora, that's going to be OK.

Reply:
For clarity, it does turn out that `ShellCheck` is in EPEL, it's just to old for our usage.

Reply:
started discussion to see if we could get a newer version in EPEL: https://src.fedoraproject.org/rpms/ShellCheck/pull-request/1